.github/workflows/cancel.yml

```diff
@@ -30,7 +30,12 @@ jobs:
     name: "Cancel duplicate workflow runs"
     runs-on: ubuntu-latest
     steps:
-      - uses: apache/airflow-cancel-workflow-runs@953e057dc81d3458935a18d1184c386b0f6b5738
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v2
```

Comment:
In build_wheels.yml we use checkout@master, why use checkout@v2 here?

Reply:
checkout@v2 is better - GitHub might introduce breaking changes in v3, so it's wise to change it in build_wheels.yml as well.

Reply:
ah thanks. Done!